dags/operators/dataproc_operator.py

```diff
@@ -0,0 +1,63 @@
+from airflow import AirflowException
+
+from airflow.contrib.operators.dataproc_operator import DataprocClusterCreateOperator
+
+class DataprocClusterCreateOperator(DataprocClusterCreateOperator):
```

Comment:
This should probably be treated as an Airflow plugin like the [custom Databricks operator](https://github.com/mozilla/telemetry-airflow/blob/master/plugins/moz_databricks.py) or the [`moz_emr` plugins](https://github.com/mozilla/telemetry-airflow/blob/master/plugins/moz_emr_plugin.py). It's nice to separate the logic into a separate module, plus the testing infrastructure is there if you want to write tests.

Reply:
just testing things out for now